### PR TITLE
Haxe language server in vscode Suggestion

### PR DESCRIPTION
**Pull request by Cheemsandfriends**
This is mostly for people who don't know about vsCode and it's a small suggestion with the haxe language support server with the small notes and bla bla bla
small change but good? idk xd

### CODE REVIEW

**Comment by Cheemsandfriends** [comment]
> 
> 
> I don't really like how this is presented It's like you're forcing everybody to use VScode
Hm, You have any suggestion? I know there's Flash Dev, Haxe thing, VS Code... But I thought in that, Maybe add a new cathegory?

File: README.md [hunk 1]
```diff
@@ -30,17 +30,22 @@ IF YOU WANT TO COMPILE THE GAME YOURSELF, CONTINUE READING!!!
 
 First, you need to install Haxe and HaxeFlixel. I'm too lazy to write and keep updated with that setup (which is pretty simple). 
 1. [Install Haxe 4.1.5](https://haxe.org/download/version/4.1.5/) (Download 4.1.5 instead of 4.2.0 because 4.2.0 is broken and is not working with gits properly...)
-2. [Install HaxeFlixel](https://haxeflixel.com/documentation/install-haxeflixel/) after downloading Haxe
+2.  You can [Install HaxeFlixel](https://haxeflixel.com/documentation/install-haxeflixel/) after downloading Haxe or you can download separately openfl, lime and hxcpp
```
Comment (Steviegt6) [inline README.md:33]: Unnecessary.

File: README.md [hunk 2]
```diff
@@ -30,17 +30,22 @@ IF YOU WANT TO COMPILE THE GAME YOURSELF, CONTINUE READING!!!
...
+lime 7.7.0
+hxcpp 4.1.15
+vshaxe
 flixel-addons
 flixel-ui
 hscript
 newgrounds
 ```
 So for each of those type `haxelib install [library]` so shit like `haxelib install newgrounds`
+(Downloading the openfl, lime and hxcpp libraries with each specific version and vshaxe can make you use the Haxe [language server](https://marketplace.visualstudio.com/items?itemName=nadako.vshaxe) in Visual Studio Code, You can't use another type of versions without failing your cache, making it very slow and tedious.)
```
Comment (Steviegt6) [inline README.md:48]: This is incoherent. Rephrase it as a suggestion for those specifically using VS Code/VS Codium.

File: README.md (hunk 1 above)
Comment (Steviegt6) [reply README.md:33]: What?

**Pull request closed without merging**